README.md

```diff
@@ -390,6 +390,12 @@ by the CI system.
 ## Builds
 * https://travis-ci.org/Sage-Bionetworks/Agora
 
+## Deployment for New Data
+* First, update data version in `data-manifest.json` in [Agoara Data Manager](https://github.com/Sage-Bionetworks/agora-data-manager/). ([example](https://github.com/Sage-Bionetworks/agora-data-manager/commit/d9006f01ae01b6c896bdc075e02ae1b683ecfd65))
```

Comment:
Small typo: "Agoara Data Manager" to "Agora Data Manager"

Reply:
Fixed. Thank you.